### PR TITLE
Update issue template to say IBI Group instead of Conveyal

### PR DESCRIPTION
**Pull request by evansiroky**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

Updates the issue template to say IBI Group and link to IBI Group's website.

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]

**Review by landonreed — APPROVED** [review]

**Review by landonreed — COMMENTED** [review]
@evansiroky, I'm wondering if we should include an email address for getting in contact for potential customers.

cc @ritesh-warade-ibigroup

**Comment by ritesh-warade-ibigroup** [comment]
yes. include my IBI email

**Comment by evansiroky** [comment]
Marking as blocked as this is now dependent on #653.

**Comment by evansiroky** [comment]
New mailto link ready for review.

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-_**NOTE:** This issue system is intended for reporting bugs and tracking progress in software development. Although this software is licensed with an open-source license, any issue opened here may not be responded to in a timely manner. [Conveyal](https://www.conveyal.com) is unable to provide technical support for custom deployments of this software unless your company has a support contract with us. Please remove this note when creating the issue._
+_**NOTE:** This issue system is intended for reporting bugs and tracking progress in software development. Although this software is licensed with an open-source license, any issue opened here may not be responded to in a timely manner. [IBI Group](https://www.ibigroup.com/) is able to provide technical support for custom deployments of this software. Please contact [Ritesh Warade](mailto:ritesh.warade@ibigroup.com?subject=Data%20Tools%20inquiry%20via%20GitHub&body=Name:%20%0D%0AAgency/Company:%20%0D%0ABest%20date/time%20for%20a%20demo/discussion:%20%0D%0ADescription%20of%20needs:%20) if your company is interested in opening a support contract with us. Please remove this note when creating the issue._
```
Comment (binh-dam-ibigroup) [inline .github/issue_template.md:1]: I would say "... if your company **or organization** is interested ..." to make it more generic.

**Review by binh-dam-ibigroup — APPROVED** [review]
Approval stands with the suggestion to use the term "organization".

File: .github/issue_template.md (hunk 1 above)
Comment (br648) [reply .github/issue_template.md:1]: Perhaps "dealt with" instead of "responded to"

**Review by br648 — APPROVED** [review]
One minor suggestion, but approving.

**Pull request merged**